diff --git a/SPECS/disruptor.spec b/SPECS/disruptor.spec
index 0a33137..eac8cb1 100644
--- a/SPECS/disruptor.spec
+++ b/SPECS/disruptor.spec
@@ -12,12 +12,20 @@ Source1:       https://repo1.maven.org/maven2/com/lmax/%{name}/%{version}/%{name
 BuildRequires: maven-local-openjdk8
 BuildRequires: mvn(junit:junit)
 BuildRequires: mvn(org.apache.felix:maven-bundle-plugin)
+AutoReqProv:   no
+Requires:      (java-headless or java-11-headless)
+Requires:      javapackages-filesystem
+Provides:      mvn(com.lmax:disruptor) = 3.4.4
+Provides:      mvn(com.lmax:disruptor:pom:) = 3.4.4
+Provides:      osgi(com.lmax.disruptor) = 3.4.4
 
 %description
 A High Performance Inter-Thread Messaging Library.
 
 %package javadoc
 Summary:       Javadoc for %{name}
+AutoReqProv:   no
+Requires:      javapackages-filesystem
 
 %description javadoc
 This package contains javadoc for %{name}.
